specs/phase1/beacon-chain.md

```diff
@@ -993,9 +1038,19 @@ def process_epoch(state: BeaconState) -> None:
     process_reveal_deadlines(state)
     process_slashings(state)
     process_final_updates(state)
+    process_phase_1_final_updates(state)
+```
+
+#### Phase 1 final updates
+
+```python
+def process_phase_1_final_updates(state: BeaconState) -> None:
     process_custody_final_updates(state)
     process_online_tracking(state)
     process_light_client_committee_updates(state)
+
+    # Update current_epoch_start_shard
+    state.current_epoch_start_shard = get_start_shard(state, state.slot)
```

Comment:
`process_epoch` happens before `state.slot` is incremented in the state transition so I think this is going to update the start shard to be the shard shard of `epoch_start_slot - 1` for the coming epoch.

Does it need to be the following?

```python
    state.current_epoch_start_shard = get_start_shard(state, state.slot + 1)
```

Reply:
Yes, you're right. 👍